transform/macros/repo_events_by_period.sql

```diff
@@ -0,0 +1,11 @@
+{% macro repo_event_by_period(event, period) -%} 
+    SELECT
+        repo_id, 
+        repo_name, 
+        DATE_TRUNC('{{ period }}', event_date) AS date_month, 
```

Comment:
Instead of `date_month` we should use the period to get `date_{{period}}` as the name of the field.